test/functional/feature_blockfilterindex_prune.py

```diff
@@ -26,9 +26,7 @@ def run_test(self):
         assert_greater_than(len(self.nodes[0].getblockfilter(self.nodes[0].getbestblockhash())['filter']), 0)
         # Mine two batches of blocks to avoid hitting NODE_NETWORK_LIMITED_MIN_BLOCKS disconnection
```

Comment:
This is wrong

Reply:
Ok, actually it doesn't matter: https://github.com/bitcoin/bitcoin/pull/23365#discussion_r745466739